gytheio-content-handlers/gytheio-content-handler-s3/src/test/java/org/gytheio/content/handler/s3/S3ContentReferenceHandlerImplTest.java

```diff
@@ -202,12 +216,133 @@ private void testFileOpsImpl()
         }
         catch (IOException ioe)
         {
-        	System.err.println(ioe.getMessage());
+        	logger.error(ioe.getMessage());
+        }
+
+        if (cleanup)
+        {
+            // delete the S3 object
+            handler.delete(reference);
+
+            assertFalse(handler.isContentReferenceExists(reference));
         }
+    }
+    
+    /**
+     * Tests: putFile (+ delete for cleanup)
+     */
+    @Test
+    public void testPutFile() throws IOException
+    {
+        String envVar;
+
+        envVar = getVar("pf_start_size");
```

Comment:
This might be easier to read if the `getVar` method used generics on an default value argument?

Reply:
Done - thanks for the feedback.